src/main/webapp/pages/projects/project_linelist.html

```diff
@@ -11,162 +11,6 @@
-                    cancel: /*[[#{linelist.tour.edit.content.cancel}]]*/ "EDIT CANCEL"
-                  }
-                },
-                search: {
-                  title: /*[[#{linelist.tour.search.title}]]*/ "SEARCH TITLE",
-                  content: /*[[#{linelist.tour.search.content}]]*/ "SEARCH CONTENT"
-                },
-                filterCounts: {
-                  title: /*[[#{linelist.tour.filterCounts.title}]]*/ "FILTER TITLE",
-                  content: {
-                    intro: /*[[#{linelist.tour.filterCounts.content.intro}]]*/ "FILTER DESC",
-                    example: /*[[#{linelist.tour.filterCounts.content.example}]]*/ "FILTER EXAMPLE"
-                  }
-                },
-                export: {
-                  title: /*[[#{linelist.tour.export.title}]]*/ "EXPORT TITLE",
-                  content: /*[[#{linelist.tour.export.content}]]*/ "EXPORT CONTENT"
-                },
-                import: {
-                  title: /*[[#{linelist.tour.import.title}]]*/ "IMPORT TITLE",
-                  content: {
-                    intro: /*[[#{linelist.tour.import.content.intro}]]*/ "IMPORT CONTENT INTRO",
-                    li1: /*[[#{linelist.tour.import.content.li1}]]*/ "IMPORT CONTENT LI1",
-                    li2: /*[[#{linelist.tour.import.content.li2}]]*/ "IMPORT CONTENT LI2"
-                  }
-                },
-                select: {
-                  title: /*[[#{linelist.tour.select.title}]]*/ "SELECT TITLE",
-                  content: /*[[#{linelist.tour.select.content}]]*/ "SELECT CONTENT"
-                },
-                selectCounts: {
-                  title: /*[[#{linelist.tour.selectCounts.title}]]*/ "SELECT COUNTS TITLE",
-                  content: /*[[#{linelist.tour.selectCounts.content}]]*/ "SELECT COUNTS CONTENT"
-                },
-                cart: {
-                  title: /*[[#{linelist.tour.cart.title}]]*/ "CART TITLE",
-                  content: /*[[#{linelist.tour.cart.content}]]*/ "CART CONTENT"
-                }
-              },
-              addToCart: /*[[#{linelist.addToCart}]]*/ "ADD TO CART",
-              importBtn: {
-                text: /*[[#{linelist.importBtn.text}]]*/ "IMPORT STUFF"
-              },
-              editing: {
-                undo: {
-                  full: /*[[#{linelist.editing.undo.full}]]*/ "The sample has been updated with a new value",
-                  empty: /*[[#{linelist.editing.undo.empty}]]*/ "The samples metadata was removed"
-                }
-              },
-              selected: {
-                none: /*[[#{linelist.selected.none}]]*/ "No samples selected",
-                one: /*[[#{linelist.selected.one}]]*/ "One sample selected",
-                multiple: /*[[#{linelist.selected.multiple}]]*/ "Many samples selected"
-              },
-              error: {
-                message: /*[[#{linelist.error.message(${project.getLabel()})}]]*/ "DANGER WILL ROBINSON"
```

Comment:
This is going to be an issue.  Here the message requires the project name passed into it.  He could do that in the javascript code pretty easily, but we need to make sure that it is done. e.g.:
```javascript
__("linelist.error.message").replace("{0}, project.label); // or something like that.
```

Reply:
I fixed this, project.getLabel returns the name and the project hash in window.PAGE has name attribute to I accessed it that way.